include/curl-session.h

```diff
@@ -78,7 +78,8 @@ class CurlSession : public CurlCallbacks
     }
 
 
-    void addFormFile(const std::string& key, const std::string& filePath, const std::string& mimeType);
+    void addFormFile(const char* key, const char* filePath, const char* mimeType);
```

Comment:
Purpose of comment to keep code consistent if it does not contradict performance requirements. If we do C++ style we shall keep C++ style where possible. C style looks like inconsistency.